atest/testdata/standard_libraries/string/get_matching_lines.robot

```diff
@@ -64,6 +67,27 @@ Get Lines Matching Regexp With Case-Insensitive
     Test Get Lines Matching Regexp    ${INPUT}    (?i).*LINE    Third line
     Test Get Lines Matching Regexp    ${INPUT}    .*LINE.*    ${EMPTY}
 
+Get Lines Containing Regexp Matching Two Words
+    Test Get Lines Containing Regexp    ${INPUT}    more|row    One more row here
+
```

Comment:
Oh, this is clearly a pair of the above new test. I don't think either of them are really needed. They seem to basically test that Python handles `|` correctly in regexps.